learning_resources/etl/edx_shared.py

```diff
@@ -37,19 +38,35 @@ def get_most_recent_course_archives(
     if s3_prefix is None:
         s3_prefix = "courses"
     try:
-        course_tar_regex = rf".*/{s3_prefix}/.*\.tar\.gz$"
-        most_recent_export_date = next(
+        log.info("Getting recent archives from %s with prefix %s", bucket, s3_prefix)
+        course_tar_regex = (
+            rf"{s3_prefix}/.*\.tar\.gz$"
+            if override_base_prefix
+            else rf".*/{s3_prefix}/.*\.tar\.gz$"
+        )
+        most_recent_export_file = next(
             reversed(  # noqa: C413
                 sorted(
                     [
                         obj
-                        for obj in bucket.objects.filter(Prefix="20")
```

Comment:
Can "20" be a constant at the top of the file to make it clearer what it is? Or maybe even a environment variavle

Reply:
Moved to top of file as a constant